app/javascript/components/widget-controller.jsx

```diff
@@ -70,7 +70,7 @@ export const WidgetController = ({ client, cityName }) => {
         if (error) {
           // eslint-disable-next-line
           console.error(error);
-          return <DisconnectedDisplay cityName={cityName} error={error} />;
+          return <DisconnectedDisplay cityName={cityName} loading={loading} />;
```

Comment:
Why have you changed it to loading? Don't we display "loading"  only when it's loading?

Reply:
Another product of my awful rebase - thanks for pointing it out, I'll  restore the files and lines that didn't need any editing.